mlip/2021_06_09_vertex_ai_predictions/deploy_model_vertexai.ipynb

```diff
@@ -0,0 +1,502 @@
+{
```

Comment:
Would include some intro with some context (f.e. only for tensorflow in this notebook)